README.md

```diff
@@ -1,104 +1,128 @@
-```
-$ git clone https://github.com/pravega/flink-connectors.git
-$./gradlew install
-```
+## Flink Connector Build Instructions
 
-### Building the Samples
-Use the built-in gradle wrapper to build the samples.
-```
-$ ./gradlew build
-...
-BUILD SUCCESSFUL
-```
+To execute Flink connector examples, follow the below steps to build and publish artifacts from 
+source to local Maven repository:
 
-### Distributing (Flink Samples)
-#### Assemble
-Use gradle to assemble a distribution folder containing the Flink programs as a ready-to-deploy uber-jar called `pravega-flink-examples-0.1.0-SNAPSHOT-all.jar`.
 ```
-$ ./gradlew installDist
-...
-$ ls -R flink-examples/build/install/pravega-flink-examples
-bin	lib
+$ git clone --recursive https://github.com/pravega/flink-connectors.git
+$ cd flink-connectors
+$ ./gradlew clean install
+```
 
-flink-examples/build/install/pravega-flink-examples/bin:
-run-example
+For more information, please visit [Flink connector Git repository](https://github.com/pravega/flink-connectors). 
 
-flink-examples/build/install/pravega-flink-examples/lib:
-pravega-flink-examples-0.1.0-SNAPSHOT-all.jar
-```
+## Pravega Samples Build Instructions
 
-#### Upload
-The `upload` task makes it easy to upload the sample binaries to your cluster.  First, configure Gradle
-with the address of a node in your cluster.   Edit `~/.gradle/gradle.properties` to specify a value for `dcosAddress`.
+Finally, we need to build the code of the examples. To this end, use the built-in gradle wrapper 
+to build the samples as follows:
 
 ```
-$ cat ~/.gradle/gradle.properties
-dcosAddress=10.240.124.164
+$ git clone https://github.com/pravega/pravega-samples.git
+$ cd pravega-samples
+$ ./gradlew clean installDist
 ```
+To ease their execution, most examples can be run either using the gradle wrapper (gradlew) or 
+scripts. The above gradle command automatically creates the execution scripts that can be found
+under:
 
-Then, upload the samples to the cluster.  They'll be copied to `/home/centos` on the target node.
 ```
-$ ./gradlew upload
+pravega-samples/standalone-examples/build/install/pravega-standalone-examples/bin
 ```
 
-## Flink Samples
+There is a Linux/Mac script and a Windows (.bat) script for each separate executable.
 
-### Anomaly Detection
-A Flink streaming application for detecting anomalous input patterns using a finite-state machine.
+# Proposed Roadmap
 
-_See the [anomaly-detection/](https://github.com/pravega/pravega-samples/tree/master/anomaly-detection) directory for more information._
+We propose a roadmap to proceed with the execution of examples based on their complexity:
+1. [Pravega features](https://github.com/pravega/pravega-samples/tree/master/standalone-examples): First step to understand the basics of 
+Pravega and exercise the concepts presented in the documentation. 
+2. [Pravega scenarios](https://github.com/pravega/pravega-samples/tree/master/standalone-examples): These examples will demonstrate the 
```

Comment:
here (2) and below (3)  `examples to demonstrate/show`

Reply:
Done.